src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure implements AreaCalculator, Drawable {
```

Comment:
```suggestion
public class Rectangle extends Figure {
```
let's remove it everywhere except Figure class